HoloPipelines/cleanUp.py

```diff
@@ -0,0 +1,30 @@
+import os
+import shutil
+
+if not os.path.exists("medicalScans"):
```

Comment:
Some issues here:
1. the if / else conditionals in this file contain much duplicate code. If you really need if/else, just do the parts that differ in the conditional section, and have the shared code run in all cases.
1. Why does a cleanUp file _create_ directories? If you want to make sure the directories exist, you should probably check/create them in the setUp parts of testing, where you also do the downloads. Otherwise you don't know if someone ran the cleanUp script before (which they won't, in case of the first run)
1. In fact i just saw that you create the directories in `pipelineController.main` anyway, so no need to do it here. Just delete them I guess.

Reply:
'cleanUp.py' removed as clean up is done within the fixture in test_pipelines.py